ui/nuxeo-filters-behavior.js

```diff
@@ -241,7 +241,7 @@ export const FiltersBehavior = {
    * Checks if the document can be tagged.
    */
   isTaggable(doc) {
-    return !this.isProxy(doc) && !doc.isVersion && this.hasPermission(doc, 'Write');
+    return !this.isProxy(doc) && !doc.isVersion && this.hasPermission(doc, 'WriteVersion');
```

Comment:
I'm wondering why do we need  `WriteVersion` to tag a document? :thinking: I would expect us to need `WriteProperties` though, given that the tags are written to the `nxtag:tags`  field.

Reply:
I agree. I will most likely leave this one out of the scope of this ticket. If it is reported as problem in the future it should be an easy fix.

Reply:
I agree.